judge-d-server/src/main/java/dev/hltech/dredd/domain/contracts/JpaServiceContractsRepository.java

```diff
@@ -29,6 +29,14 @@ public ServiceContracts persist(ServiceContracts serviceContracts) {
         return ofNullable(entityManager.find(ServiceContracts.class, new ServiceContracts.ServiceContractsId(name, version)));
     }
 
+    @Override
+    public String getService(String name) {
+        return entityManager
+            .createQuery("select distinct o.id.name from " + ServiceContracts.class.getName() + " o where o.id.name = :name", String.class)
```

Comment:
Select name of the service filtering all available services with given name of the service?

Do you plan to verify if the given name exists?